miniserver/config.js

```diff
@@ -0,0 +1,81 @@
+require('dotenv').config()
```

Comment:
Some unused fields and constants should be cleaned up here

Reply:
Done